src/xrUICore/Windows/UIFrameWindow.cpp

```diff
@@ -318,7 +318,7 @@ void CUIFrameWindow::draw_tile_line(Frect rect, int i, bool b_horz, Fvector2 con
     }
 }
 
-void CUIFrameWindow::draw_tile_rect(Frect rect, int i, Fvector2 const& ts)
+void CUIFrameWindow::draw_tile_rect(Frect& rect, int i, Fvector2 const& ts)
```

Comment:
ARE YOU SURE that there's no potential error?

Reply:
The project is builded and played normally. If you need to check something extra, I will check it out.